src/Adapter/Carrier/CommandHandler/AddCarrierHandler.php

```diff
@@ -57,17 +57,27 @@ public function __construct(
     public function handle(AddCarrierCommand $command): CarrierId
     {
         $carrier = new Carrier();
+        // General information
         $carrier->name = $command->getName();
         $carrier->grade = $command->getGrade();
         $carrier->url = $command->getTrackingUrl();
         $carrier->position = $command->getPosition();
         $carrier->active = $command->getActive();
         $carrier->delay = $command->getLocalizedDelay();
 
+        // Shipping information
+        $carrier->shipping_handling = $command->isShippingHandling();
```

Comment:
I'm not sure this naming is understandable 🤔 What is shipping handling? Can the wording be improved to understand it better?

Reply:
I agree with you guyes that the naming are not appropriate here.
But I don't agree with you @Hlavtox about the "should be `get`" because that not the additional fee amount here but the status of if we add additional fees for this carrier.
What about `hasAdditionalHandlingFee`?

Reply:
@boherm Sorry, you are right, I am dumb, it's an on/off switch, there should be `IS`. :-)